dts/riscv/it8xxx2.dtsi

```diff
@@ -559,6 +572,14 @@
 			status = "disabled";
 			label = "ADC_0";
 			#io-channel-cells = <1>;
+			pinctrls = <&pinmuxi 0 IT8XXX2_PINMUX_FUNC_1   /* ADC0*/
```

Comment:
I thought the consensus was that we were going to use pinctrl-0, pinctrl-1, etc. in the pinmux discussions I've been present at during the weekly API meetings. This seems a bit different -- @mnkp any comment here?

Reply:
Done